highlight/highlight
# Create pending invites tab

## Summary
We want to allow the User Management page to show "Pending Invites" depending on the tab the user selects.
[Figma designs
](https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?type=design&node-id=8287-324021&t=SPkMxFB7uP9tkc89-0)

This PR is a step towards these changes:
- Create nav bar that will allow user to select "Pending invites" tab"
  - "members" tab shown by default
  - "invites" tab is disabled, but navigatable through `/w/1/team/invites`
- Updates to the "Invite user" button
  - Update copy and add icon
  - Move out of the top level header down to the tab content
- Rename page from "User management" to "Members"

## How did you test this change?
1. Select "Workspace settings" option in the settings menu
2. Switch the top level tabs on the left to ensure they still work
3. Select the "Members" tab to see the updated team page
- [ ] Confirm "Members" tab is active
- [ ] Confirm "Pending Invites" tab is disabled
- [ ] Can still update "Auto join" settings
- [ ] Can still invite users
- [ ] Can still manage members
4) Navigate to "Pending Invites" page (`/w/1/team/invites`)
- [ ] Confirm "Pending Invites" tab is selected
- [ ] "Coming soon" content is sufficient to display
- [ ] Can switch back to "Members" tab


https://github.com/highlight/highlight/assets/17744174/a2574ff3-e038-4850-a7b6-4c0d44d45d5d


## Are there any deployment considerations?
N/A

## Considerations
- Is "Coming soon" content sufficient to display in production, or should we hide behind variable?
- Do we want to include hotkeys (`m` and `p`) for switch tabs?
  - There will be numbers in the places of hotkeys, so may need a different way to educate. So left them out for now.
